src/sentry/middleware/ratelimit.py

```diff
@@ -91,6 +92,9 @@ def process_view(self, request: Request, view_func, view_args, view_kwargs) -> R
                     request.will_be_rate_limited = True
                     enforce_rate_limit = getattr(view_class, "enforce_rate_limit", False)
                     if enforce_rate_limit:
+                        logger.info(
```

Comment:
This won't show up properly in google logging. You'll need to do something like
```
logger.info("sentry.api.ratelimit.exceeded", extra={"key": request.rate_limit_key})
```

see:   https://github.com/getsentry/sentry/blob/3cfa5c961f86ae2054bc34388896f88ce4b0208b/src/sentry/integrations/vsts/webhooks.py#L44 as an example